### ISSUE TITLE
pylint install failing - posssibly due to zip-date for asteroid 2.5.6

### ISSUE THREAD

**Issue by clawsl** [David-OConnor/pyflow#131]
Hi,
After a few working installs, I tried adding pylint as dev dependency 
and got the following result on my windows 10 machine using pyflow 0.3.0

``` 
pyflow install -d pylint
Found lockfile
Installing lazy_object_proxy 1.6.0 ...
Installing mccabe 0.6.1 ...
Installing pylint 2.8.3 ...
Added a console script: epylint
Added a console script: pylint
Added a console script: pyreverse
Added a console script: symilar
Installing isort 5.9.1 ...
Added a console script: isort
Added a console script: isort-identify-imports
Installing astroid 2.5.6 ...
thread 'main' panicked at 'running setup.py bdist_wheel in folder "__pypackages__\\3.8\\lib\\astroid-2.5.6". Py path: "__pypackages__\\3.8\\.venv/Scripts\\python": Traceback (most recent call last):
  File "setup.py", line 10, in <module>
    setup(version=__version__)  # pylint: disable=undefined-variable
  File "C:\dev\py_om_graphav\__pypackages__\3.8\.venv\lib\site-packages\setuptools\__init__.py", line 145, in setup
    return distutils.core.setup(**attrs)
  File "C:\Users\klaus.fleck\AppData\Roaming\pyflow\python-3.8.0\lib\distutils\core.py", line 148, in setup
    dist.run_commands()
  File "C:\Users\klaus.fleck\AppData\Roaming\pyflow\python-3.8.0\lib\distutils\dist.py", line 966, in run_commands
    self.run_command(cmd)
  File "C:\Users\klaus.fleck\AppData\Roaming\pyflow\python-3.8.0\lib\distutils\dist.py", line 985, in run_command
    cmd_obj.run()
  File "C:\dev\py_om_graphav\__pypackages__\3.8\.venv\lib\site-packages\wheel\bdist_wheel.py", line 254, in run
    wf.write_files(archive_root)
  File "C:\dev\py_om_graphav\__pypackages__\3.8\.venv\lib\site-packages\wheel\wheelfile.py", line 124, in write_files
    self.write(path, arcname)
  File "C:\dev\py_om_graphav\__pypackages__\3.8\.venv\lib\site-packages\wheel\wheelfile.py", line 135, in write
    zinfo = ZipInfo(arcname or filename, date_time=get_zipinfo_datetime(st.st_mtime))
  File "C:\Users\klaus.fleck\AppData\Roaming\pyflow\python-3.8.0\lib\zipfile.py", line 362, in __init__
    raise ValueError('ZIP does not support timestamps before 1980')
ValueError: ZIP does not support timestamps before 1980
', src\install.rs:363:21
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by David-OConnor** [comment]
Thanks for the report! Will try to get it fixed.

**Comment by tanj** [comment]
This is fixed in master as of 3f64bf4b0a0405aaef31a2606dbe6aefecd3f0d8

@David-OConnor it might be time for another release with the pull request I just issued https://github.com/David-OConnor/pyflow/pull/134

**Comment by David-OConnor** [comment]
Awesome. Agree on the release.
